### ISSUE TITLE
Validation swallows real errors, when they occur within a validator.

### ISSUE THREAD

**Issue by JoshMcCullough** [aurelia/validation#228]
Reproduce like this:

```
ValidationGroup.prototype.test = function()
{
	this.passes((newValue, threshold) => {
		let x = null;

		x.y = "This causes a null reference error.";

		return threshold;
	});
	
	return this;
}
```

If you try this, you will get this in the console: 

```
Uncaught (in promise) Error: Unexpected behavior: a validation-rules-collection should always fulfil(…)(anonymous function) 
@ validation-property.js:79
```

The message is pretty unclear, too, but at least it should emit the actual error, if possible.

**Comment by PWKad** [comment]
Thanks for submitting this / commenting on this.  At this time we are closing this because we have completely re-written and are deprecating the previous feature set.  If you feel this should be re-opened please feel free to review [this blog post](http://blog.durandal.io/2016/05/03/aurelias-new-validation-and-testing-capabilities/) and submit either again on this repository, or on the new [validatejs bridge repository](https://github.com/aurelia/validatejs)

Thanks again!
